superset/utils/screenshots.py

```diff
@@ -17,12 +17,14 @@
 from __future__ import annotations
 
 import logging
+from datetime import datetime
+from enum import Enum
 from io import BytesIO
-from typing import TYPE_CHECKING
+from typing import Optional, TYPE_CHECKING
```

Comment:
Nit: we should not use `Optional` anymore, always just `from __future__ import annotations` and then `| None`.